lua/toggleterm/commandline.lua

```diff
@@ -41,6 +41,8 @@ function M.parse(args)
         local key, value = arg[1], arg[2]
         if key == "size" then
           value = tonumber(value)
+        elseif key == "go_back" then
+          value = tonumber(value) ~= 0
```

Comment:
Why use `0` and `1` here rather than the string true and false, seems clearer to just have an actual boolean even though that would be converted to a string and would need to be changed back to a boolean 🤷🏿‍♂️ not a big deal though